packages/core/src/contracts/tcr/eip20.ts

```diff
@@ -30,7 +42,10 @@ export class EIP20 extends BaseWrapper<EIP20Contract> {
    * @param spender address to approve as spender of tokens
    * @param numTokens number of tokens to approve for spender to spend on user's behalf
    */
-  public async approveSpender(spender: EthAddress, numTokens: BigNumber): Promise<TwoStepEthTransaction> {
+  public async approveSpender(spender: EthAddress, numTokens: BigNumber): Promise<MultisigProxyTransaction> {
```

Comment:
The user of this library will expect all mutations to work the same way. Might as well give `transfer` the multisig treatment too

Reply:
You're right, will update, though the dapp currently uses `transfer` for transferring from logged-in user *to* the multisig so we still need to support non-multisig. I'm going to update comments and variable names to clearly distinguish between `tcrViaMultisig`/`tokenViaMultisig` vs. non-multisig-instantiated ones, and library user needs to keep track of which kind they've created.

Reply:
Actually will update all of the functions in here, including `getBalance` etc., to default to the multisig address if this is a multisig instance.